Log.cpp

```diff
@@ -89,9 +91,9 @@ std::string FormatPacket(const OP2Internal::Packet& packet)
 }
 
 
-void Log(const char* string)
+void Log(const std::string& message)

```

Comment:
Probably slightly better to split this change into a separate commit. I like it though.